mmhuman3d/core/conventions/keypoints_mapping/openpose.py

```diff
@@ -3,12 +3,13 @@
 Openpose mainly supports 25 and 135 now, 118 convention can be found in
 https://github.com/vchoutas/smplify-x/issues/152#issuecomment-923715702.
 
-Openpose_curated_fits can be found in 
+OPENPOSE_137_KEYPOINTS can be found in 
 https://github.com/vchoutas/expose
 
 - OPENPOSE_25_KEYPOINTS:  body(25)
 - OPENPOSE_118_KEYPOINTS: body(25) + hand(42) + face(51)
 - OPENPOSE_135_KEYPOINTS: body(25) + hand(40) + face(70)
+- OPENPOSE_137_KEYPOINTS: body(27) + hand(40) + face(70)
```

Comment:
It will be more helpful to compare `OPENPOSE_KEYPOINTS` and `OPENPOSE_137` with existing conventions. What are the detailed differences? e.g. `OPENPOSE_KEYPOINTS` vs `OPENPOSE_137`?

Reply:
More description added.

Reply:
Add the corner case to this [issue](https://github.com/open-mmlab/mmhuman3d/issues/83)